web/src/settings_components.ts

```diff
@@ -1525,3 +1525,31 @@ export function create_group_setting_widget({
 
     return pill_widget;
 }
+
+export function set_expires_on_text($elem: JQuery, valid_to_text: string): void {
+    if ($elem.val() === "custom") {
+        $elem.parent().find(".input-time-valid-till").hide();
+        set_custom_expires_on_text($elem, valid_to_text);
+    } else {
+        $elem.parent().find(".input-time-valid-till").show();
+        $elem.parent().find(".input-time-valid-till").text(valid_to_text);
+    }
+}
+
+export function set_custom_expires_on_text($elem: JQuery, valid_to_text: string): void {
+    $elem.parent().find(".custom-input-time-valid-till").text(valid_to_text);
```

Comment:
This can just be moved to the prior function and call that function in all cases.

Also, this commit needs to update all of the places these functions are called in the invite.ts module. Otherwise this commit breaks the app/tests.